docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.8'
+
+services:
+  postgres:
+    image: postgres:13
+    environment:
+      POSTGRES_DB: mydatabase
```

Comment:
şifreler .env dosyasında okunmalı. hem local hem prod için .prod.env .local.env dosyaları oluşturup orada tutabilirsin
bkx: Docker .env